whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -222,7 +226,7 @@ private RdapObject getRdapObject(final String requestUrl,
             case INETNUM:
             case INET6NUM:
                 rdapResponse = createIp(rpslObject);
-                rdapResponse.getRdapConformance().add("cidr0");
+                rdapResponse.getRdapConformance().add(RdapConformance.CIDR_0.getValue());
```

Comment:
Add *all* extensions here?

Reply:
we must include cidr0 in all the responses, not just in INET6NUM:
https://bitbucket.org/nroecg/nro-rdap-profile/raw/v1/nro-rdap-profile.txt

4.1.
   The service MUST implement the "cidr0" extension (registered with
   IANA), so the string literal "cidr0" MUST be included as an
   rdapConformance value in **each response.**

Reply:
after converting the object from RPSL to Rdap we are calling mapCommons, so we are still adding all the extensions but the FLAT_MODEL. 
We need CIDR_0 in all the responses so I will move to mapCommons in order to have CIRD_0, LEVEL_0 and PROFILE_0 in all the responses

'''
   Each response MUST include an rdapConformance element.  The service
   MUST conform with RFC 7483 [RFC7483], so "rdap_level_0" will be
   included as an rdapConformance value in each response.

   To signify conformance with this specification, the string literal
   "nro_rdap_profile_0" MUST be included as an rdapConformance value in
   each response.

   The service MUST implement the "cidr0" extension (registered with
   IANA), so the string literal "cidr0" MUST be included as an
   rdapConformance value in each response.
'''